ttlafterfinish to beta (#26738)
kubernetes/website: content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

<!-- overview -->

{{< feature-state for_k8s_version="v1.21" state="beta" >}}

The TTL controller provides a TTL (time to live) mechanism to limit the lifetime of resource
objects that have finished execution. TTL controller only handles
{{< glossary_tooltip text="Jobs" term_id="job" >}} for now,
and may be expanded to handle other resources that will finish execution,
such as Pods and custom resources.

> This feature is currently beta and enabled by default, and can be disabled via 

Reviewer: nits (these don't block a merge):
```suggestion
This feature is beta and enabled by default. You can disable it using the
```
